File: llm_outputs/octocoder/GuruSamples/php/substitute-algorithm_refactored.php
function foundPerson(array $people): string
{
    foreach ($people as $person) {
        if ($person === 'Don') {
            return 'Don';
        }

        if ($person === 'John') {
            return 'John';
        }

        if ($person === 'Kent') {
            return 'Kent';
        }
    }

    return '';
}

File: results/llm_outputs/octocoder/GuruSamples/java/substitute-algorithm_refactored.java
String foundPerson(String[] people) {
    for (String person : people) {
        if ("Don".equals(person)) {
            return "Don";
        }
        if ("John".equals(person)) {
            return "John";
        }
        if ("Kent".equals(person)) {
            return "Kent";
        }
    }
    return "";
}
